sources/packages/web/src/components/aest/students/StudentLoanBalancePartTime.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <body-header-container :enableCardView="true">
+    <template #header>
+      <body-header
+        title="Part-Time monthly loan balance"
+        subTitle="Balance of part-time Canada Student Loan outstanding between the student and NSLSC at any given time.
+        This amount is updated once per month, and affects a student's eligibility for future funding, as a student 
+        cannot exceed the maximum limit in total outstanding balance."
+      />
+    </template>
+    <content-group>
+      <toggle-content :toggled="!studentLoanBalanceDetails.length">
+        <DataTable
+          :value="studentLoanBalanceDetails"
+          class="p-m-4"
+          :paginator="true"
+          :rows="pageLimit"
+          :rowsPerPageOptions="PAGINATION_LIST"
+        >
+          <Column field="balanceDate" header="Date">
+            <template #body="slotProps">
+              <span>
+                {{ dateOnlyLongString(slotProps.data.balanceDate) }}
+              </span>
+            </template>
+          </Column>
+          <Column field="cslBalance" header="CSLP Balance">
+            <template #body="slotProps">
+              <span>
+                {{ formatCurrency(slotProps.data.cslBalance) }}
+              </span>
+            </template></Column
+          >
+        </DataTable>
+      </toggle-content>
+    </content-group>
+  </body-header-container>
+</template>
+<script lang="ts">
+import { ref, onMounted, defineComponent } from "vue";
+import { StudentLoanBalanceService } from "@/services/StudentLoanBalanceService";
+import {
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  DEFAULT_PAGE_NUMBER,
+} from "@/types";
+import { useFormatters } from "@/composables";
+import { StudentMonthlyLoanBalanceDetailAPIOutDTO } from "@/services/http/dto";
+
+export default defineComponent({
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
+    const page = ref(DEFAULT_PAGE_NUMBER);
+    const pageLimit = ref(DEFAULT_PAGE_LIMIT);
```

Comment:
the DEFAULT_PAGE_LIMIT value is 10 and we are planning to show only the last 12 records, is that fine to have 10 as a limit and 2 in the next page or show all 12? also as per the screenshot we show only 5 values in the page, is that the right limit?

Reply:
I would say, unless it is very necessary we shouldn't modify the default behavior.